### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by TiRaMisu1024** [tjiiv-cprg/EPro-PnP#64]
1. can I simply use EPro-PnP for other two-stage pose estimation network?
2. if I wanna to do so , can I just code 'from ops.pnp.epropnp import EProPnP6DoF' and epropnp = EProPnP6DoF(...)? or what else should I do ?

Thank you very much, I am looking forward to your guidance.

**Comment by Lakonik** [comment]
Yes, you can plug EPro-PnP's solver into other two-stage network during inference (which may be faster for large batches) or use it to re-train the original network (which requires slightly modifying the original network for w2d prediction). You can refer to the [demo code](https://github.com/tjiiv-cprg/EPro-PnP/blob/main/demo/fit_identity.ipynb) for the usage.
